apstools/devices/lakeshore_controllers.py

```diff
@@ -44,7 +44,7 @@ class LakeShore336_LoopControl(PVPositionerSoftDoneWithStop):
     # readback is provided by PVPositionerSoftDoneWithStop
 
     setpoint = FormattedComponent(
-        EpicsSignalWithRBV, "{prefix}OUT{loop_number}:SP", put_complete=True
+        EpicsSignal, "{prefix}OUT{loop_number}:SP", put_complete=True
```

Comment:
I think we should reverse this, it's better to have the RBV, and it should work with the fix in the `PVPositionerSoftDone`.

Reply:
I'll make the change and test it.

Reply:
passes locally